_assets/js/lesson_plans.js

```diff
@@ -6,104 +6,104 @@
   root.lessons = {
     1: {
       0: {
-        startingText: "Writing in Markdown is not that hard!",
```

Comment:
Ah, these changes need to be made in a new `lessons/_es/lesson_plans.js` location.

Reply:
Should be fixed now ☺️